apps/openmw/mwgui/statswindow.cpp

```diff
@@ -240,6 +240,16 @@ namespace MWGui
             if (value.getBase() < 100) {
                 setSkillProgress(nameWidget, value.getProgress(), parSkill);
                 setSkillProgress(valueWidget, value.getProgress(), parSkill);
+            } else {
+                nameWidget->setUserString("Visible_SkillMaxed", "true");
```

Comment:
What if you set your skill below 100 again? The "Maximum proficiency reached" message should not stay visible.

Also, please respect our code style, that is:
```c++
if (...)
{
}
```
not
```c++
if (...) {
}
```